app/src/androidTest/java/de/westnordost/streetcomplete/data/osm/mapdata/RelationDaoTest.kt

```diff
@@ -79,44 +85,56 @@ class RelationDaoTest : ApplicationDbTestCase() {
 
     @Test fun getAll() {
         val e1 = rel(1, members = listOf(RelationMember(NODE, 0, "bla")))
-        val e2 = rel(2, members = listOf(
-            RelationMember(NODE, 0, "bla"),
-            RelationMember(WAY, 1, "blub"),
-        ))
-        val e3 = rel(3, members = listOf(
-            RelationMember(RELATION, 3, "one"),
-            RelationMember(WAY, 4, "two"),
-        ))
-        dao.putAll(listOf(e1,e2,e3))
+        val e2 = rel(
+            2,
+            members = listOf(
+                RelationMember(NODE, 0, "bla"),
+                RelationMember(WAY, 1, "blub"),
+            )
+        )
+        val e3 = rel(
+            3,
+            members = listOf(
+                RelationMember(RELATION, 3, "one"),
+                RelationMember(WAY, 4, "two"),
+            )
+        )
+        dao.putAll(listOf(e1, e2, e3))
         assertEquals(
             listOf(e1, e2),
-            dao.getAll(listOf(1,2,4)).sortedBy { it.id }
+            dao.getAll(listOf(1, 2, 4)).sortedBy { it.id }
         )
         assertEquals(
             listOf(e1, e2, e3),
-            dao.getAll(listOf(1,2,3)).sortedBy { it.id }
+            dao.getAll(listOf(1, 2, 3)).sortedBy { it.id }
         )
     }
 
     @Test fun deleteAll() {
         dao.putAll(listOf(rel(1), rel(2), rel(3)))
-        assertEquals(2, dao.deleteAll(listOf(1,2,4)))
+        assertEquals(2, dao.deleteAll(listOf(1, 2, 4)))
         assertNotNull(dao.get(3))
         assertNull(dao.get(1))
         assertNull(dao.get(2))
     }
 
     @Test fun getAllForElement() {
         val e1 = rel(1, members = listOf(RelationMember(NODE, 0, "bla")))
-        val e2 = rel(2, members = listOf(
-            RelationMember(NODE, 0, "bla"),
-            RelationMember(WAY, 1, "blub"),
-        ))
-        val e3 = rel(3, members = listOf(
-            RelationMember(RELATION, 3, "one"),
-            RelationMember(WAY, 4, "two"),
-        ))
-        dao.putAll(listOf(e1,e2,e3))
+        val e2 = rel(
+            2,
+            members = listOf(
+                RelationMember(NODE, 0, "bla"),
+                RelationMember(WAY, 1, "blub"),
+            )
+        )
+        val e3 = rel(
+            3,
+            members = listOf(
+                RelationMember(RELATION, 3, "one"),
+                RelationMember(WAY, 4, "two"),
+            )
+        )
```

Comment:
Which rule is this? I find the way it was before more readable.

Reply:
Again, the `indent` rule ([`9c2f170`](https://github.com/streetcomplete/StreetComplete/pull/3649/commits/9c2f170cfb7c138d01501e0c7ccbecc54f41b511#diff-1da3edd1bb0a07d6737c2859252bff1d67eb85cdbccc9219024ac5fdb485e6b8)).